### PR TITLE
Add new fields to product table

### PR DESCRIPTION
**Pull request by kayobruno**

### CODE REVIEW

**Comment by kayobruno** [comment]
[Add module to manage Stock Movements](https://trello.com/c/aGkBsCPh/28-add-module-to-manage-stock-movements)

File: app/Models/Product.php [hunk 1]
```diff
@@ -34,4 +36,12 @@ public function merchant(): BelongsTo
     {
         return $this->belongsTo(Merchant::class, 'id');
     }
+
+    public function formatPrice(): string
```
Comment (WebertySilveira) [inline app/Models/Product.php:40]: Seria interessante separarmos essas ferramentas em helpers para serem utilizados em todas as views?
O próprio laravel tem uns helpers de formatação que poderiam se encaixar nesse momento:
ex: https://laravel.com/docs/10.x/helpers#method-number-format

**Review by WebertySilveira — COMMENTED** [review]
There is no need for separation yet, but it is interesting to consider.

File: app/Models/Product.php (hunk 1 above)
Comment (kayobruno) [reply app/Models/Product.php:40]: I think it's not a good idea, as creating a helper with multiple responsibilities isn't that interesting. But you gave me another idea, that is sanitizing data on input and not on output.

**Pull request merged**